packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -330,6 +330,7 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
 
     // If this is a controlled component then no state change should take place.
     if (selectedKey !== undefined || selectedKeys !== undefined) {
+      this._onChange(event, options, index, checked, multiSelect);
```

Comment:
I wonder if this fix will be considered a "breaking" change for some? Just wondering if this change needs to be communicated during release...

Reply:
Since the behavior being replaced wasn't the expected one (that was broken 5 days ago), I don't think anyone is relying on it.

Reply:
Yes, I didn't know it was a recent regression until the PR description was updated. 😄